tests/functional/gcs/test_user_credential.py

```diff
@@ -0,0 +1,77 @@
+import json
+
+from globus_sdk._testing import get_last_request, load_response
+from globus_sdk.services.gcs import UserCredentialDocument
```

Comment:
I just noticed this import. To me, this means that we're missing the name exported in `src/globus_sdk/__init__.py`.

It should be sufficient to append it to [the list in the init generator](https://github.com/globus/globus-sdk-python/blob/574bf8e22913d9dc8eb465bc647b8749a1841fd1/src/globus_sdk/_generate_init.py#L106) and then run that script.